File: Toony Timepieces/Toony Timepieces.js
from flask import Flask
import mysql.connector

app = Flask(__name__)

# Database connection configuration
db_config = {
    'host': 'localhost',
    'user': 'root',
    'password': 'your_password',
    'database': 'toony_timepieces'
}

# Establish database connection
db = mysql.connector.connect(**db_config)

@app.route('/')
def home():
    return "Welcome to Toony Timepieces E-commerce Platform!"

if __name__ == '__main__':
    app.run(debug=True)

File: Toony Timepieces/Toony Timepieces.py
from flask import Flask, jsonify, request
import mysql.connector

app = Flask(__name__)

# Create a connection to the database
db = mysql.connector.connect(
    host="localhost",
    user="root",
    password="",
    database="toony_timepieces"
)

# API endpoint to get products
@app.route('/api/products', methods=['GET'])
def get_products():
    cursor = db.cursor(dictionary=True)
    cursor.execute("SELECT * FROM products")
    products = cursor.fetchall()
    cursor.close()
    return jsonify(products)

# Start the server
if __name__ == '__main__':
    app.run(debug=True)
